### PR TITLE
docs(borealis): DOC-517 update helm chart and parameters

### PR DESCRIPTION
**Pull request by 317brian**
Pages/sections to review:

Clarifying "environment" cause I had to think through it while writing: 
* https://deploy-preview-1136--armory-docs.netlify.app/borealis#terminology

New deployment target workflow. Aurora reuses the same snippets so no need to review both. Will port the changes over to Aurora completely after we've reviewed them for Borealis.
* https://deploy-preview-1136--armory-docs.netlify.app/borealis/quick-start/borealis-org-get-started/#prepare-your-deployment-target
* https://deploy-preview-1136--armory-docs.netlify.app/borealis/quick-start/borealis-org-get-started/#migrate-to-the-new-rna

UI pages:
Config UI: https://deploy-preview-1136--armory-docs.netlify.app/borealis/ui/borealis-configuration-ui/
Leftover multi-target that I merged into this PR: https://deploy-preview-1136--armory-docs.netlify.app/borealis/ui/borealis-status-ui/

To do:

Need to create generic secrets engine pages since the existing ones might confuse people. (They mention Spinnaker specifically)

### CODE REVIEW

**Comment by 317brian** [comment]
K8s page content hidden for now in HTML comments

File: content/en/armory-enterprise/installation/aurora-install.md [hunk 1]
```diff
@@ -84,6 +85,7 @@ Register your Armory Enterprise environment so that it can communicate with Armo
    - `read:infra:data`
    - `exec:infra:op`
    - `read:artifacts:data`
+   - `connect:agentHub`
```
Comment (fieldju) [inline content/en/armory-enterprise/installation/aurora-install.md]: `connect:agentHub` is the only required scope the rest of these are dropped now.

File: content/en/borealis/not-a-real-page.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+   *  If you set `kubernetes.enableClusterAccountMode` to true, the RNA creates a service account, cluster role, and cluster role binding. These are applied to the RNA, and the RNA registers itself as a target deployment cluster for Project Borealis.
+
+
+    The following example includes the `--create-namespace` option:
+
+    ```bash
+    helm upgrade --install armory-rna armory/remote-network-agent \
+        --set agentIdentifier=<target-cluster-name> \
+        --set clientId=<clientID-for-rna> \
+        --set clientSecret=<clientSecret-for-rna> \
```
Comment (fieldju) [inline content/en/borealis/not-a-real-page.md]:
```suggestion
        --set clientId=encrypted:k8s!n:rna-client-credentials!k:client-id \
        --set clientSecret=encrypted:k8s!n:rna-client-credentials!k:client-secret \
```

File: content/en/borealis/not-a-real-page.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+4. Install the Helm chart. Keep the following in mind when you install the RNA:
+
+   * You can create the `armory-rna` namespace before running the `helm upgrade --install` command or include the `--create-namespace` option.
+
+   * The `agentIdentifier` option is the name that is used to refer to the deployment target cluster in the Status UI, your deployment file, and other places, so use a descriptive name.
+
+   * Armory recommends storing and using the values for`clientID` and `clientSecret` as secrets. You can use either a secret engine directly or environment variables to store and pass the encrypted values.
+
+     * If you use a secret engine, pass the encrypted value for the parameter in the command, such as `'encrypted:k8s!n:rna-client-credentials!k:client-secret'`. Armory supports the following secret engines: Vault, Kubernetes secrets, encrypted GCS buckets, encrypted S3 buckets, and AWS Secrets Manager. See the documentation for your secrets engine for the format of the encrypted value.
```
Comment (fieldju) [inline content/en/borealis/not-a-real-page.md]: Can we add the recommended k8s example and hide some of these details

```shell
kubectl --namespace armory-rna create secret generic rna-client-credentials --type=string --from-literal=client-secret=xxx-yyy-ooo --from-literal=client-id=zzz-ooo-qqq
```

File: content/en/borealis/not-a-real-page.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+If you installed an older version of the RNA that used the Helm chart stored in `armory/aurora`, migrate to the new version by updating the Helm chart that is used. You do not need to do this if you are installing the RNA for the first time.
+
+
+1. Navigate to the [Machine to Machine Client Credentials](https://console.cloud.armory.io/configuration/credentials) page in the Cloud Console.
+2. Update the permissions for the RNA you are using for your deployment targets. They must have the `connect:agentHub` scope. This permission is used to connect the new RNA endpoint.
+3. Verify that you are in the correct Kubernetes context. The context should be for the deployment target cluster.
+4. Change the Helm chart that is being used.
+
+   If you installed the RNA by providing your `clientID` and `clientSecret` in plaintext, now is a good time to update those values to use a secret engine.
```
Comment (fieldju) [inline content/en/borealis/not-a-real-page.md]: ex:

```shell
kubectl --namespace armory-rna create secret generic rna-client-credentials --type=string --from-literal=client-secret=xxx-yyy-ooo --from-literal=client-id=zzz-ooo-qqq
```

**Comment by fieldju** [comment]
On this page: https://deploy-preview-1136--armory-docs.netlify.app/borealis/borealis-requirements/

image

`agents.cloud.armory.io` is being replaced with `agent-hub.cloud.armory.io` with the same description.

But its not used by Spinnaker

**Comment by fieldju** [comment]
I'd maybe nix these whole sections
image
image

And just say if you have advanced use cases to cover such as proxy configuration, adding custom annotations, labels, or environment variables please see the [remote network agents values.yaml](https://github.com/armory-io/remote-network-agent-helm-chart/blob/master/values.yaml) and the [official helm documentation for using a values file](https://helm.sh/docs/chart_template_guide/values_files/) for all advanced configuration options.

and maybe the getting started guide is literally just covering the base case

```shell
# Create the K8s secret for your OIDC client credentials
kubectl --namespace armory-rna create secret generic rna-client-credentials --type=string --from-literal=client-secret=<clientID-for-rna> --from-literal=client-id=<clientSecret-for-rna>
# Optionally Add Armory Chart repo, if you haven't
helm repo add armory https://armory.jfrog.io/artifactory/charts
# Update repo to fetch latest armory charts
helm repo update
# Install or Upgrade armory rna chart
helm upgrade --install armory-rna armory/remote-network-agent \
    --set clientId='encrypted:k8s!n:rna-client-credentials!k:client-id' \
    --set clientSecret='encrypted:k8s!n:rna-client-credentials!k:client-secret' \
    --namespace armory-rna
```

File: content/en/borealis/not-a-real-page.md (hunk 3 above)
Comment (fieldju) [reply content/en/borealis/not-a-real-page.md]: We might just nix this whole block about secrets and just add using k8s secrets as a step and have a quick comment about there being more there or something

**Comment by fieldju** [comment]
@stephenatwell has mentioned a desire to hide this option `--set kubernetes.enableClusterAccountMode=<false|true>`, it defaults to true and a user can opt out if they dig into advanced options

**Comment by fieldju** [comment]
image

Should `status ui` say cloud console for consistency?

File: content/en/borealis/not-a-real-page.md (hunk 3 above)
Comment (stephenatwell) [reply content/en/borealis/not-a-real-page.md]: yes, let's please be prescriptive about this. Really I want the quick starts to be step by step instructions with no options.

**Comment by stephenatwell** [comment]
> advanced use cases to cover such as proxy configuration, adding custom annotations, labels, or environment variables please see t

- [ ] @brian-armory, can we not include stuff like `proxy configuration` in the quick start guide, but instead full them out into a separate help topic called something like `Advanced RNA options`, and cover them there?
 
Also, Justin is right, no need to document `--set kubernetes.enableClusterAccountMode=<false|true>` at all.

**Comment by stephenatwell** [comment]
> image
> 
> Should `status ui` say cloud console for consistency?

considering we don't use the phrase 'cloud console' anywhere, I don't think so...... but it also should say 'status ui' since it's not part of that. I'd say something like:

Got to the agents page on the configuration tab of https://console.cloud.armory.io. or 'go to the agents page on https://console.cloud.armory.io/configuration'

**Comment by fieldju** [comment]
> > image
> > Should `status ui` say cloud console for consistency?
> 
> considering we don't use the phrase 'cloud console' anywhere, I don't think so...... but it also should say 'status ui' since it's not part of that. I'd say something like:
> 
> Got to the agents page on the configuration tab of https://console.cloud.armory.io. or 'go to the agents page on https://console.cloud.armory.io/configuration'

@stephenatwell 

I've seen the UI mentioned as Cloud Console mentioned 2x

image
image

and in this instance referenced as Status UI.

I don't really care all that much as to what we call it, more that its consistant.

**Comment by 317brian** [comment]
That specific screenshot is a typo. It should be Agents page in the Configuration UI. 

The way I'm referring it to the docs is

- The whole UI is Cloud Console

- The admin/config portion is Configuration UI since that's what the tab is labeled as.

- The deployments part is called the Armory Deployments Status UI (or Status UI) for short since the tab is called Armory Deployments and what it shows is the status. The Status UI name first showed up in the msg that the CLI returns when you trigger a deployment, so that's why I started using it.

I don't have strong feelings about what we name them, but they do need names or else it's really hard for me to write about them.

**Comment by 317brian** [comment]
Reviews are done. Will merge tomorrow morning to coincide with the code changes going out.

**Pull request merged**